hal/api/PwmOut.h

```diff
@@ -67,6 +67,10 @@ class PwmOut {
         core_util_critical_section_exit();
     }
 
+    /** To Free up pwm pin.
+     */
+    virtual ~PwmOut();
```

Comment:
Why is dtor ``virtual`` ? I don't see any virtual method within this class, it's not intended to be a base class.